pkg/ccl/changefeedccl/cdctest/nemeses.go

```diff
@@ -582,10 +608,22 @@ func addColumn(a fsm.Args) error {
 			`there are less than %d columns.`, ns.maxTestColumnCount)
 	}
 
-	if _, err := ns.db.Exec(fmt.Sprintf(`ALTER TABLE foo ADD COLUMN test%d STRING DEFAULT 'x'`,
-		ns.currentTestColumnCount)); err != nil {
-		return err
+	// Some of the time, add a column of an enum type.
+	if ns.enumCount > 0 && rand.Intn(4) < 1 {
```

Comment:
I think this is fine, but if we add many more that work like this it could get hard to have a sense for what the actual mix of events is since this depends both on the randomness above and the previous ordering of events.